champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/sword/Ride.java

```diff
@@ -131,6 +132,29 @@ public boolean canUse(Player player) {
         return true;
     }
 
+    @EventHandler
+    public void onHorseDamage(CustomDamageEvent event) {
+        if (!(event.getDamager() instanceof Player damager)) return;
+        if (!(event.getDamager() instanceof Horse damagee)) return;
+        HorseData data = horseData.get(damager);
+        if (data != null && data.getHorse().equals(damagee)) {
+            // If the horse belongs to the player who is damaging it, we ignore the event.
```

Comment:
Would you not want to cancel the event so horse owners can't kill their own horse spanwned by this ability?

Reply:
That'd only apply if the Clans plugin was on though

Reply:
valid point, fix it ezrn

Reply:
weve been down this road a week ago I don't know how to do it without connecting to clans

Reply:
if damgee instanceof Tameable tamedHorse
if(tamedHorse.getOwner().equals(damager))
cancel event

Reply:
yeah I know how to do that I don't know how to do it for allies

Reply:
champions doesnt have a concept of allies in this version, so you can let clans handle that